### PR TITLE
[zh]Sync release notes for Istio 1.13.7

### PR DESCRIPTION
**Pull request by jzhupup**
- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by zirain — CHANGES_REQUESTED** [review]
should not contains 1.13.8

File: content/zh/news/releases/1.13.x/announcing-1.13.7/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+linktitle: 1.13.7
+subtitle: 补丁发布
+description: Istio 1.13.7 补丁发布。
+publishdate: 2022-08-01
+release: 1.13.7
+aliases:
+    - /zh/news/announcing-1.13.7
+---
+
+此版本包含针对 [CVE-2022-31045](/zh/news/security/istio-security-2022-005/#cve-2022-31045) 和错误修复以提高稳定性。我们建议用户安装此版本而不是 Istio 1.13.6，因为 Istio 1.13.7 包含上述 CVE 修复。
```
Comment (windsonsea) [inline content/zh/news/releases/1.13.x/announcing-1.13.7/index.md:12]:
```suggestion
此版本包含针对 [CVE-2022-31045](/zh/news/security/istio-security-2022-005/#cve-2022-31045) 所做的修复和其他漏洞修复，以提高稳健性。我们建议用户安装此版本来替代未包含上述 CVE 修复的 Istio 1.13.6。
```

File: content/zh/news/releases/1.13.x/announcing-1.13.7/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+这个版本说明描述了 Istio 1.13.6 和 Istio 1.13.7 之间的区别。
```
Comment (windsonsea) [inline content/zh/news/releases/1.13.x/announcing-1.13.7/index.md:13]:
```suggestion
这个发布说明描述了 Istio 1.13.6 和 Istio 1.13.7 之间的区别。
```

File: content/zh/news/releases/1.13.x/announcing-1.13.7/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+其中包括 9 个安全修复程序。如果你在本地使用 Go，我们建议你升级到这个较新的 Go 版本。
```
Comment (windsonsea) [inline content/zh/news/releases/1.13.x/announcing-1.13.7/index.md:15]: A clause is missing:
FYI, [Go 1.18.4 has been released](https://groups.google.com/g/golang-announce/c/nqrv9fbR0zE),
which includes 9 security fixes. We recommend you to upgrade to this newer Go version if you are using Go locally.

**Review by windsonsea — CHANGES_REQUESTED** [review]
Respect comes from your carefulness.

File: content/zh/news/releases/1.13.x/announcing-1.13.7/index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+release: 1.13.7
+aliases:
+    - /zh/news/announcing-1.13.7
+---
+
+此版本包含针对 [CVE-2022-31045](/zh/news/security/istio-security-2022-005/#cve-2022-31045) 所做的修复和其他漏洞修复，以提高稳健性。我们建议用户安装此版本来替代未包含上述 CVE 修复的 Istio 1.13.6。
+这个发布说明描述了 Istio 1.13.6 和 Istio 1.13.7 之间的区别。
+
+仅供参考，[Go 1.18.4 已发布](https://groups.google.com/g/golang-announce/c/nqrv9fbR0zE)，
+其中包括 9 个安全修复。如果你在本地使用 Go，我们建议你升级到这个较新的 Go 版本。
```
Comment (windsonsea) [inline content/zh/news/releases/1.13.x/announcing-1.13.7/index.md:16]:
```suggestion
其中包括 9 个安全修复。如果您在本地使用 Go，我们建议您升级到这个较新的 Go 版本。
```

Comment (windsonsea) [reply content/zh/news/releases/1.13.x/announcing-1.13.7/index.md:16]: 您 is preferred in Istio localization.

**Review by windsonsea — APPROVED** [review]

**Review by zirain — APPROVED** [review]

**Pull request merged**